app/importers/envirotech/analysis_link_data.rb

```diff
@@ -42,7 +42,7 @@ def process_article_info(article_hash)
 
       article[:source] = model_class.source[:code]
 
-      article[:id] = Utils.generate_id(article, %i(source_id source))
+      article[:id] = article[:source_id]
```

Comment:
Good to know that it is okay to use the relation id for ES id, i think we can now improve the `update` method in `Indexable`! will take a look at this later.